rpc/execution_pool.go

```diff
@@ -8,27 +8,27 @@ import (
 	"github.com/JekaMas/workerpool"
 )
 
-const (
-	threads        = 100
-	requestTimeout = 0 //10 * time.Second
-)
-
 type SafePool struct {
 	executionPool *atomic.Pointer[workerpool.WorkerPool]
 	fastPath      bool
+	timeout       *atomic.Pointer[time.Duration]
```

Comment:
if timeout was 0 then fastPath was true, if we then change timeout to non-zero value, fastPath incorrectly will be kept true.
I believe we need `fastPath` atomic and change `ChangeTimeout` `Change*`. Maybe have 1 RWMutex instead 2 atomic values?

Reply:
fastpath is dependent upon size and not timeout I think